variables.tf

```diff
@@ -35,6 +35,34 @@ variable "cluster_settings" {
   }
 }
 
+variable "cluster_service_connect_defaults" {
+  description = "Configures a default Service Connect namespace"
+  type        = map(string)
+  default     = {}
+}
+
+################################################################################
+# CloudWatch Log Group
+################################################################################
+
+variable "create_cloudwatch_log_group" {
+  description = "Determines whether a log group is created by this module for the cluster logs. If not, AWS will automatically create one if logging is enabled"
+  type        = bool
+  default     = true
+}
+
+variable "cloudwatch_log_group_retention_in_days" {
+  description = "Number of days to retain log events. Default retention - 90 days"
+  type        = number
+  default     = 90
```

Comment:
```suggestion
  default     = null
```

Reply:
we always provide a default value in our modules to ensure logs are eventually purged and avoid wasting $$ - 90 was selected based on heuristics but we can look at changing the default if necessary. However, we should absolutely avoid `null` here which would retain logs indefinitely